lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -209,6 +168,51 @@ class _WebGlRenderer implements _GlRenderer {
     context.restore();
   }
 
+  // Vertex shader transforms pixel space [Vertices.positions] to
```

Comment:
suggestion: here in and in the next method we can use doc comments (///)

Reply:
Done.